.travis.yml

```diff
@@ -3,7 +3,11 @@ language: java
 jdk:
   - oraclejdk8
 
-install: mvn clean install
+install:
+  - mvn -s .travis.maven.settings.xml -version -B
```

Comment:
Should we try to include -fae (fail at end), so that the CI build tries to do as much as possible, which could allow to uncover more than the first issue in a push?